tests/test_esm_runscripts/test_filedicts.py

```diff
@@ -155,3 +182,49 @@ def test_mv(fs):
         "/work/ollie/pgierz/some_exp/run_20010101-20010101/work/unit.24"
     )
 
+
+# ===
+# Tests for SimulationFile.ln() method
+# + check if source exists
+# + check if destination is written
+# + check if destination already exists or a symlink
+# + check for self pointing
+# + check if source path does not exist
+# + check if destination path does not exist
+# ===
+def test_ln_iscallable(simulation_file):
+    assert isinstance(simulation_file.ln, Callable)
+
+def test_ln_links_files_correctly(simulation_file, fs):
```

Comment:
Question: does the order of the fixtures you use as function arguments matter? Or is that arbitrary?